tests/fixtures.py

```diff
@@ -49,6 +49,44 @@ def basic_specs_fixture():
                             'mount': '/lib/a',})
     _write('service', 'service-a', {'image': 'service/a'})
 
+def specs_fixture_with_depends():
+    _write('bundle', 'bundle-a', {'description': 'Bundle A', 'apps': ['appa']})
+    _write('bundle', 'bundle-b', {'description': 'Bundle B', 'apps': ['appb']})
+    _write('app', 'appa', {'repo': '/tmp/app-a',
```

Comment:
Can we call the repo `repo-a` or something? Would make the tests much more clear